### PR TITLE
fix mysql column comment has no effects problem

### PR DESCRIPTION
**Pull request by sysoft**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by sushantdhiman** [comment]
This will need tests and first a PR to master

File: lib/dialects/mysql/query-generator.js [hunk 1]
```diff
@@ -340,6 +340,10 @@ const QueryGenerator = {
     if (attribute.primaryKey) {
       template += ' PRIMARY KEY';
     }
+    
+    if (attribute.comment) {
+      template += ` COMMENT ${this.escape(attribute.comment)}`;
```
Comment (SimonSchick) [inline lib/dialects/mysql/query-generator.js]: Isn't this missing quotes?

Comment (sysoft) [reply lib/dialects/mysql/query-generator.js]: already added quotes in escape function

**Comment by sushantdhiman** [comment]
This is already supported in `master`, so closing. Once you have added tests please ping me.

**Pull request closed without merging**